додав розмітку переваг

nazoferon01 opened the pull request.

nazoferon01 committed:
додав розмітку переваг

vladpolischchuk commented on src/partials/advantages.html:
```diff
@@ -0,0 +1,35 @@
+                        <use href="./images/icons.svg#advant1"></use>
+                    </svg>
+                </div>
+                <h3 class="advantages__title">721</h3>
```
має бути `class="section-title"`

nazoferon01 replied:
виправив

vladpolischchuk commented on src/partials/advantages.html:
```diff
@@ -0,0 +1,35 @@
+                        <use href="./images/icons.svg#advant2"></use>
+                    </svg>
+                </div>
+                <h3 class="advantages__title">16kg</h3>
```
має бути `class="section-title"`

nazoferon01 replied:
виправив

vladpolischchuk commented on src/partials/advantages.html:
```diff
@@ -0,0 +1,35 @@
+                        <use href="./images/icons.svg#advant3"></use>
+                    </svg>
+                </div>
+                <h3 class="advantages__title">84</h3>
```
має бути `class="section-title"`

nazoferon01 replied:
виправив

nazoferon01 committed:
виправив назву класу

section-title

nazoferon01 committed:
Додатково приховав заголовок секції

vladpolischchuk commented on src/sass/layouts/_advantages.scss:
```diff
@@ -0,0 +1,14 @@
+/*прихований заголовок*/
```
Зроби будь ласка паршал: sass/utils/_visuallly-hidden.scss 
Та підєднай його в паршал _module.scss

nazoferon01 replied:
створив та підєднав

nazoferon01 committed:
створив паршал visually-hidden

Зробив паршал: sass/utils/_visuallly-hidden.scss
Та підєднав його в паршал _module.scss
